src/syscall/syscall_libc.go

```diff
@@ -212,6 +212,13 @@ func Truncate(path string, length int64) (err error) {
 	return
 }
 
+//go:linkname syscall_fcntl runtime/runtime.fcntl
```

Comment:
I think you meant `runtime.fcntl`?

```suggestion
//go:linkname syscall_fcntl runtime.fcntl
```